test/blackbox-tests/test-cases/watching/watching-eager-concurrent-build-command.t

```diff
@@ -26,6 +27,13 @@ Demonstrate that we can run "dune build" while the watch server is running.
   $ dune build
   Success
 
+Demonstrate that a warning is displayed when extra arguments are passed to
+"dune build", since those arguments will be ignored.
+  $ dune build --auto-promote 2>&1 | sed 's/pid: [0-9]*/pid: PID/g'
+  Warning: Your build request is being forwarded to a running Dune instance
+  (pid: PID). Note that certain command line arguments may be ignored.
+  Success
+
```

Comment:
Can you add an additional test case that shows that no such warning is printed if no arguments are ignored?

Reply:
Does the command on line 26 of this file satisfy that?